src/packet/buffer_tx.rs

```diff
@@ -7,6 +7,7 @@ use crate::rtp::{ExtensionValues, MediaTime, Rid, RtpHeader, SeqNo, Ssrc};
 use super::{CodecPacketizer, PacketError, Packetizer, QueueSnapshot};
 use super::{MediaKind, QueuePriority};
 
+#[derive(Clone)]
```

Comment:
I don't want these cloned.

Reply:
I don't have another solution

Reply:
Storing an index into `PacketizingBuffer::queue` should work, no?

Reply:
I intentionally stored up to 60 representative sizes to have a size-diversity you may not find in the current buffer while also removing the need to keep this in sync with it.

Reply:
@davibe can we use the SeqNo?

If each entry have `HashMap<Pt, BTreeMap<usize, SeqNo>>`, the selection can be quick and retrieving the SeqNo not quite as quick, but maybe enough?

Would still be nice to keep it inside buffer_tx though.

Reply:
With that solution I would do it in the buffer_tx so it's easier to keep in sync, otherwise you may store a seq_no that's no longer there

Reply:
@davibe @k0nserv maybe we can use `Rc` and `Weak`?

I don't want `Arc` inside str0m since it has locking, but `Rc` should be lock free right?

Reply:
`Arc` doesn't contain a lock, it does use atomic though. Using `Rc` probably makes the whole `Rtc` instance `!Send` which will prevent it from being used with Tokio

Reply:
Ah yes. We do want Send Sync.